WordPressShared.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressShared"
-  s.version       = "1.1.0"
+  s.version       = "1.1.1-beta.4"
```

Comment:
this is because it took so long for us to review your changes, so it's a 100% on us, but we've already released `  `1.1.1-beta.4`, so this would need to be `.5`. sorry!

Reply:
👍 updating